tests/bdd/steps/h_api.py

```diff
@@ -0,0 +1,30 @@
+import re
+
+from h_matchers import Any
+from httpretty import httpretty
+
+from tests.bdd.step_context import StepContext
+
+
+class HAPIContext(StepContext):
```

Comment:
We now try to send requests to H as we have configured launches. We need to accept these gracefully or the tests fail.

Reply:
Aha, yes! Quick crash course for me in `httpretty`. I think I object to the _naming_ of that library as it sounds like a linting tool, not a mocking tool, grin. I wonder if it's worth commenting the `enable` line for nitwits, or whether 99.5% of anyone versed in python should be familiar with it...

Reply:
I've added some comments to the file to say what this is all about.